mpas_analysis/sea_ice/modelvsobs.py

```diff
@@ -53,9 +52,9 @@ def seaice_modelvsobs(config, streamMap=None, variableMap=None):
     startDate = config.get('climatology', 'startDate')
     endDate = config.get('climatology', 'endDate')
     streamName = streams.find_stream(streamMap['timeSeriesStats'])
-    infiles = streams.readpath(streamName, startDate=startDate,
+    inFiles = streams.readpath(streamName, startDate=startDate,
                                endDate=endDate, calendar=calendar)
-    print 'Reading files {} through {}'.format(infiles[0], infiles[-1])
+    print 'Reading files {} through {}'.format(inFiles[0], inFiles[-1])
 
     plotsDirectory = buildConfigFullPath(config, 'output', 'plotsSubdirectory')
     obsDirectory = config.get('seaIceObservations', 'baseDirectory')
```

Comment:
I know this is not the right place to do it, but I realized recently that we should have the iceCocentration and iceThickness observational files defined in the config file, rather than hard wired here...